x/subscription/spec/02_state.md

```diff
@@ -0,0 +1,35 @@
+<!--
+order: 2
+-->
+
+# State
+
+The `x/subscription` stores the plans and subscriptions on chain:
+
+```golang
+type Plan struct {
+  id int,  // auto-increasing unique identifier
+  title string,
+  description string,
+  owner Address,  // beneficial owner of the plan
+  price Decimal,  // price in fiat currency show to user
+  fiat_currency String,  // fiat currency name
+  token_denom String,  // denomation of token
```

Comment:
how is `String` different from `string`? good to make it clear with the respect to: https://github.com/crypto-com/chain-main/issues/336

I guess it can be made more general by being called e.g. `plan_denom` or `plan_currency` (instead of `fiat_currency`) and `subscriber_denom` (instead of `token_denom`) in the case there are use cases where the plan isn't denominated in fiat?

Reply:
Use `Coin` directly now, which should be well defined concept already I think.